dll/cpl/hotplug/hotplug.h

```diff
@@ -38,7 +38,38 @@ typedef struct
     APPLET_PROC AppletProc;
 }APPLET, *PAPPLET;
 
+typedef struct _HOTPLUG_DATA
+{
+    HICON hIcon;
+    HICON hIconSm;
+    SP_CLASSIMAGELIST_DATA ImageListData;
+    HMENU hPopupMenu;
+    HWND hwndDeviceTree;
```

Comment:
Is this HOTPLUG_DATA a structure for _**one**_  device? If this is so, why make it also dependent on the hwndDeviceTree window?

Reply:
> Is this HOTPLUG_DATA a structure for _**one**_ device?

No, it's for the entire app.

Reply:
ok thanks